docker/api/requirements.txt

```diff
@@ -12,3 +12,4 @@ python-jose[cryptography]==3.3.0
 pyyaml==5.3.1
 redis==5.0.1
 uvicorn[standard]==0.13.4
+kernelci
```

Comment:
I'd suggest also mentioning the package version here.

Reply:
We need to create the versions though ;)  That's related to what I mentioned on Slack last week, we need a `pyproject.toml` file and get this uploaded to PyPI.  In the meantime, it's possible to use a Git branch e.g.:
```
kernelci@git+https://github.com/kernelci/kernelci-core@release-branch
```